src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -606,6 +606,19 @@ bool fuse_type_to_nms5(const std::shared_ptr<ov::Node>& node, const precisions_m
     }
 
     bool res = false;
+    auto type_relaxed = std::dynamic_pointer_cast<ov::op::TypeRelaxedBase>(node);
```

Comment:
do we have other similar implementations in this transformation?
In which it's also worth moving that part up

Reply:
I haven't found similar implementations in the transformation.